Makefile

```diff
@@ -219,12 +221,40 @@ help:					#: Show the help messages
 	awk '/^[-a-z]+:/' | \
 	awk -F: '{ printf("%-20s   %s\n", $$1, $$NF) }'
 
-python-clean:
-	DEBUG=1 ${PYTHON_EXECUTABLE} setup.py clean
-
-python-build:
-	DEBUG=1 CMAKE_BUILD_PARALLEL_LEVEL=${NUM_THREADS} ${PYTHON_EXECUTABLE} -m pip install -e .$(extras) --verbose
-
-python-test:
-	$(MAKE) python-build extras="[tests]"
-	DEBUG=1 ${PYTHON_EXECUTABLE} -m unittest -v
+python-venv:
+	@if [ ! -f ${PYTHON_VENV}/pyvenv.cfg ]; then \
+		${VENV} $(PYTHON_VENV); \
+	fi
+
+check-pip-version: python-venv # We need a recent pip for '-C'
+	if [ "$(PIP)" == "uv pip" ]; then \
+		exit 0; \
+	fi; \
+	source .venv/bin/activate; \
+	found=0; \
+	for pydir in $(PYTHON_VENV)/lib/python*/site-packages; do \
```

Comment:
nit  - cant we just run pip -v in the shell , and then use that to get major / minor version .

Reply:
uh, yeah I guess that would work too. For some reason I wanted to get it from a file.